num_c: 3
buffer_topic: 'laser_wall_extraction/buffer'
frame_id: 'laser_link'
clusters_topic: '~clusters'
publish_cluster_labels: False
use_overlap: False
cluster_labels_topic: '~cluster_labels'
